src/controllers/notifications.js

```diff
@@ -70,7 +71,19 @@ const model = require('../models/notifications')
 exports.post = function * postNotification () {
   const notification = yield requestUtil.validateBody(this, 'Notification')
   log.debug('Got notification: ' + JSON.stringify(notification))
-  yield model.processNotification(notification)
-
+  try {
+    yield model.processNotification(notification)
+  } catch (e) {
+    // If this receives UnacceptableExpiryError, do not respond with HTML error 422.
+    // Doing so would cause Ledger to reply with the same message, creating an infinite loop.
+    if (e instanceof UnacceptableExpiryError) {
+      log.debug('Notification handling received non-critical error: ' + e)
+      this.status = 202
```

Comment:
As discussed with Matt offline, I picked 202 because I'd like Connector to return success status code (2xx), because otherwise Ledger would resend notifications. Within all 2xx's, this one (202 - Accepted) seems to be a good semantic match, as Connector accepted the notification, even though it didn't act on it.